doc/themes/scikit-learn/layout.html

```diff
@@ -38,6 +38,14 @@
            $(this).css('background-color', '#A7D6E2');
        }
     );
+    function showMenu() {
```

Comment:
The function is called when the user clicks on the hamburger. It adds (or gets rid of) a class to the hamburger called 'responsive' which changes the display of the menu.